report/finished/chapters/introduction.tex

```diff
@@ -5,7 +5,7 @@ \subsection{Background and Motivation}
 % Why project course?
 As part of the Engineering Project course for Master's students in Computer Science, Control and Electrical Engineering at Luleå University of Technology the goal is to implement and demonstrate how the Arrowhead Framework can be used in a factory setting using autonomous ground robots. This project report aims to give an overview of the thought process, workflows and results of the project.
 
-The proposed solution is a robot which by implementing machine vision algorithms can navigate the factory floor while also being able to pick up object using it's arm and gripper. It utilizes the Arrowhead framework to integrate with the other parts of the model factory. 
+The proposed solution is a robot which by implementing machine vision algorithms can navigate the factory floor while also being able to pick up objects using it's arm and gripper. It utilizes the Arrowhead framework to integrate with the other parts of the model factory. 
 
 \subsection{Contributions}
 Our proposed solution is an example of how the Arrowhead framework is utilized in a scaled factory setting together with a ground robot. The design is small and modular making it easy for future imrovements and easy evaluation of the Arrowhead framework alongside third-party open-source solutions. Secondly we propose a solution consisting of lines and QR-codes and how the robot can fllow a path using a RGB-camera instead of the conventional sensors. With this camera solution the possibilities of navigating the factory floor increase compared to the basic conventional following solutions. 
```

Comment:
line 11: spelling error "improvements"
Secondly we propose a solution consisting of lines and QR-codes: 
- might want to mention colored line, since we could use different color for different things.

navigating the factory floor increase compared to the basic conventional following solutions. 
- what basic solutions. Might need to be mentioned.